pom.xml

```diff
@@ -158,6 +158,12 @@
       <groupId>com.axonivy.ivy.api</groupId>
       <artifactId>ivy-core-api</artifactId>
       <version>${ivy.api}</version>
+      <exclusions>
+        <exclusion>
+          <groupId>org.apache.logging.log4j</groupId>
+          <artifactId>log4j-1.2-api</artifactId>
```

Comment:
you may use `<artifactId>*</artifactId>` to avoid similar issues with other versions.